notebooks/enterprise-search/elastic-crawler-to-open-crawler-migration.ipynb

```diff
@@ -0,0 +1,555 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "89b4646f-6a71-44e0-97b9-846319bf0162",
+   "metadata": {},
+   "source": [
+    "## Hello, future Open Crawler user!\n",
+    "[![Open In Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/elastic/elasticsearch-labs/blob/main/notebooks/enterprise-search/elastic-crawler-to-open-crawler-migration.ipynb)\n",
+    "\n",
+    "This notebook is designed to help you migrate your Elastic Crawler configurations to Open Crawler-friendly YAML!\n",
+    "\n",
+    "We recommend running each cell individually in a sequential fashion, as each cell is dependent on previous cells having been run. Furthermore, we recommend that you only run each cell once as re-running cells may result in errors or incorrect YAML files.\n",
+    "\n",
+    "### Setup\n",
+    "First, let's start by making sure `elasticsearch` and other required dependencies are installed and imported by running the following cell:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "da411d2f-9aff-46af-845a-5fe9be19ea3c",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!pip install elasticsearch\n",
+    "\n",
+    "from getpass import getpass\n",
+    "from elasticsearch import Elasticsearch\n",
+    "\n",
+    "import os\n",
+    "import json\n",
+    "import yaml\n",
+    "import pprint"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "f4131f88-9895-4c0e-8b0a-6ec7b3b45653",
+   "metadata": {},
+   "source": [
+    "We are going to need a few things from your Elasticsearch deployment before we can migrate your configurations:\n",
```

Comment:
What if the user is running the Elastic Crawler with Enterprise Search on a self-managed deployment?

👇 This makes it seem like this is only relevant to Elastic Cloud Hosted users.

Reply:
@pquentin as Python client expert, I'd ask for confirmation that this approach works equally well for both :)